src/workerd/jsg/tracing-test.c++

```diff
@@ -27,7 +27,72 @@ public:
   Ref<NumberBox> getInner() { return inner.addRef(); }
 
   JSG_RESOURCE_TYPE(NumberBoxHolder) {
-    JSG_READONLY_INSTANCE_PROPERTY(inner, getInner);
+    JSG_READONLY_PROTOTYPE_PROPERTY(inner, getInner);
```

Comment:
Why change the property type here?

Reply:
Ah, I see the additional comment below about v8 treating it as modified. This will be fairly important to document.